terraso_backend/apps/story_map/views.py

```diff
@@ -87,7 +95,7 @@ def post(self, request, **kwargs):
                 extra={"user_id": user.pk, "story_map_id": str(story_map.id)},
             )
             error_message = ErrorMessage(
-                code="update", context=ErrorContext(model="StoryMap", field=NON_FIELD_ERRORS)
+                code="update", context=ErrorContext(model="StoryMap", field="new_config")
```

Comment:
Issue: The field should be related to the data the user sent, so I think in this case it should be `configuration` instead of `new_config`